config/jobs/kubernetes/sig-cloud-provider/aws/sig-aws-credentials.yaml

```
@@ -28,3 +28,18 @@ presets:
   - name: aws-cred
     secret:
       secretName: aws-credentials-607362164682
+
+# Credentials for using AWS account `k8s-infra-e2e-scale-boskos-001`. Used for scale tests in AWS.
+- labels:
+    preset-aws-credential-aws-scale-testing: "true"
+  env:
+  - name: AWS_SHARED_CREDENTIALS_FILE
+    value: /etc/aws-cred/credentials
```

Please change this to:

```suggestion
    value: /etc/aws-cred/sharedCredentialsFile
```

As per https://github.com/kubernetes/k8s.io/pull/5676